core/src/main/java/com/radixdlt/modules/DispatcherModule.java

```diff
@@ -234,7 +244,7 @@ public void configure() {
             Dispatchers.dispatcherProvider(
                 RoundQuorumReached.class,
                 (counters, event) -> {
-                  if (event.votingResult() instanceof RoundVotingResult.FormedTC) {
+                  if (event.roundQuorum() instanceof RoundQuorum.TimeoutRoundQuorum) {
```

Comment:
EDIT: I was wrong, confused by the naming. `RoundQuorumReached` is actually `RoundQuorumResolution`.

So I suggest we remove these metrics, as I think it overlaps with RoundResolutions? (which I seem to recall you've already added?)

Possibly having a separate metric for `unresolvedQuorumsReached` might be useful too? But it should likely be incremented inside the BFTQuorumAssembler.

Reply:
Yup, they overlap with `roundChanges` - I'll remove them. I don't think we need `unresolvedQuorumsReached` and it may be slightly ambiguous. E.g. would we also increment it if we receive another vote, form a timeout quorum, and ignore it because there's already a delayed TC in flight?

Reply:
hmh, those metrics are used in some tests :thinking: and it's not really easy (or I don't know how to) extract those values from a labelled `RoundChange` by a single dimension.
I've kept them, but renamed to `quorumResolutions()` labelled with bool isTimeout.

Reply:
Ahh sorry I meant `timeoutQuorumDelayedResolutions` when I said `unresolvedQuorumsReached`. I'm not sure what I was thinking when I wrote the latter haha.